build/buildAmbientTypes.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
```

Comment:
Why didn't you put this in the scripts folder? Seems like a more logic place for this transform.